src/bin/encode.rs

```diff
@@ -88,7 +88,7 @@ fn handle_path<'a>(options: &Options<'a>,
         .expect("Could not open source")
         .len();
 
-    println!("Parsing.");
+    println!("Parsingq.");
```

Comment:
Please fix this typo :)

Reply:
err... this was in my old commit 😁 Let me rebase this :)